code/implementation/docker/common/src/main/java/io/cattle/platform/docker/constants/DockerInstanceConstants.java

```diff
@@ -5,7 +5,29 @@
     public static final String FIELD_DOCKER_PORTS = "dockerPorts";
     public static final String FIELD_DOCKER_HOST_IP = "dockerHostIp";
     public static final String FIELD_DOCKER_IP = "dockerIp";
+    public static final String FIELD_DOCKER_INSPECT = "dockerInspect";
     public static final String FIELD_VOLUMES_FROM = "dataVolumesFrom";
+    public static final String FIELD_DOMAIN_NAME = "domainName";
+    public static final String FIELD_USER = "user";
+    public static final String FIELD_MEMORY = "memory";
+    public static final String FIELD_MEMORY_SWAP = "memorySwap";
+    public static final String FIELD_CPU_SHARES = "cpuShares";
+    public static final String FIELD_CPU_SET = "cpuSet";
+    public static final String FIELD_TTY = "tty";
+    public static final String FIELD_STDIN_OPEN = "stdinOpen";
+    public static final String FIELD_DIRECTORY = "directory";
+    public static final String FIELD_COMMAND = "command";
```

Comment:
We _just_ eliminated that: we have a single command parameter that is an array.
https://github.com/rancherio/cattle/pull/370